lib/core/backend/backend.ts

```diff
@@ -247,6 +251,15 @@ export class Backend {
       writable: true,
     });
 
+    Reflect.defineProperty(global, "nodeId", {
+      get: () => {
+        return this.nodeId;
+      },
+      set: () => {
+        throw new Error("nodeId is read-only");
```

Comment:
Can it just be writable: false here ?

Reply:
Nop can't define setter/getter and use writable property (I tried your suggestion first while developing)